maestro-test/src/main/kotlin/maestro/test/drivers/FakeDriver.kt

```diff
@@ -341,6 +338,14 @@ class FakeDriver : Driver {
         }
     }
 
+    override fun waitForAppToSettle(initialHierarchy: ViewHierarchy?): ViewHierarchy {
+        return ViewHierarchy(TreeNode())
```

Comment:
This should perform in the same way as Android _or_ iOS would

Reply:
what if they perform differently, should I rely on Android implementation or iOS? :)

Reply:
That is a good question. Perhaps make it iOS-like by default and then maybe we can make it configurable later if we want to